plugins/notion/src/blocksToHTML.ts

```diff
@@ -92,7 +92,38 @@ export function blocksToHtml(blocks: BlockObjectResponse[]) {
                 break
             }
             case "code":
-                htmlContent += `<pre><code class="language-${block.code.language.replace(" ", "-")}">${richTextToHTML(block.code.rich_text)}</code></pre>`
+                htmlContent += `<pre data-language="${
+                    CODE_LANGUAGE_MAPPING[block.code.language] || "JavaScript"
+                }"><code>${richTextToHTML(block.code.rich_text)}</code></pre>`
```

Comment:
```suggestion
                const language = CODE_LANGUAGE_MAPPING[block.code.language]

                if (language) {
                    htmlContent += `<pre data-language="${language}"><code>${richTextToHTML(block.code.rich_text)}</code></pre>`
                } else {
                    htmlContent += `<pre><code>${richTextToHTML(block.code.rich_text)}</code></pre>`
                }
```

For this to work, you will need to:
- encapsulate the case in curly brackets
- type `CODE_LANGUAGE_MAPPING` to `Record<string, string | null>`.